### ISSUE TITLE
Error in IMP example

### ISSUE THREAD

**Issue by deleeke** [IBAMR/IBAMR#139]
I get a runtime error when I try and run the IMP example

```

/IMP/explicit/ex0$ ./main2d input2d
 
*** Warning, This code is deprecated, and likely to be removed in future library versions! ./include/libmesh/auto_ptr.h, line 270, compiled Aug  2 2016 at 11:58:24 ***
P=00000:Program abort called in file ``../../IBAMR/lib/../src/IB/IMPInitializer.cpp'' at line 234
P=00000:ERROR MESSAGE: 
P=00000:Failed assertion: silo_writer
[deleeke-x401a:03573] *** Process received signal ***
[deleeke-x401a:03573] Signal: Aborted (6)
[deleeke-x401a:03573] Signal code:  (-6)
[deleeke-x401a:03573] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x10330)[0x7f2745432330]
[deleeke-x401a:03573] [ 1] /lib/x86_64-linux-gnu/libc.so.6(gsignal+0x37)[0x7f273ff79c37]
[deleeke-x401a:03573] [ 2] /lib/x86_64-linux-gnu/libc.so.6(abort+0x148)[0x7f273ff7d028]
[deleeke-x401a:03573] [ 3] ./main2d[0xb3b6b3]
[deleeke-x401a:03573] [ 4] ./main2d[0xafe514]
[deleeke-x401a:03573] [ 5] ./main2d[0x68eb2f]
[deleeke-x401a:03573] [ 6] ./main2d[0x42d5a3]
[deleeke-x401a:03573] [ 7] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf5)[0x7f273ff64f45]
[deleeke-x401a:03573] [ 8] ./main2d[0x42b289]
[deleeke-x401a:03573] *** End of error message ***
Aborted (core dumped)
```

Let me know if you need any more details!

**Comment by boyceg** [comment]
Fixed by #172.

**Comment by kdelee** [comment]
@boyceg FYI I don't know if you have noticed this but you can have issues automatically closed if you included "closes #(issue#)" in the commit message

https://help.github.com/articles/closing-issues-using-keywords/
